simon-says.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap"
+      rel="stylesheet"
+    />
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Simon Says!</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet"type="text/css"media="screen"href="simon-says.css"/>
+  </head>
+  <body id="body">
```

Comment:
Clase padre del juego seria este body, seguramente quieras que esté centrado y con medidas que se vean siempre bien en cualquier pantalla

Con un [display flex ](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)podés centrarlo fácilmente 
y las medidas mínimas o máximas con max y min height/width

Reply:
Sisi. Este tema justamente no llegué a verlo, gracias por anticiparme! Es lo que tenía que revisar justamente.